### PR TITLE
[Bug] fix requirements.txt for example chat-w/-pdf

### PR DESCRIPTION
**Pull request by LatentDream**
The example is missing some dependencies

To resolve this comment: https://github.com/ploomber/cloud-frontend/pull/601#pullrequestreview-2341988066

----
📚 Documentation preview 📚: https://ploomber-doc--284.org.readthedocs.build/en/284/

### CODE REVIEW

File: examples/panel/chat-with-pdf/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,137 @@
-panel
-openai
-chromadb
-pypdf
-langchain
-tiktoken
\ No newline at end of file
+aiohappyeyeballs==2.4.3
+aiohttp==3.9.3
+aiosignal==1.3.1
```
Comment (neelasha23) [inline examples/panel/chat-with-pdf/requirements.txt:3]: All the dependencies can go in a `requirements.lock.txt` file. In this file, you can add the specific package that was needed. You can see the format in the other examples

Comment (neelasha23) [reply examples/panel/chat-with-pdf/requirements.txt:3]: Yes the lock file is for debugging purposes. We can just add the specific package required to resolve the issue `langchain_community.document_loaders` in this file. I'm not sure why all the installed packages need to be mentioned in the `requirements.txt` file

Comment (neelasha23) [reply examples/panel/chat-with-pdf/requirements.txt:3]: It is mentioned in the error: `You can install it using `pip install -U langchain-community`
So just add langchain-community in this file

Comment (neelasha23) [reply examples/panel/chat-with-pdf/requirements.txt:3]: We generally freeze all requirements in the lock file and not the requirements.txt. But maybe @edublancas can suggest?

File: examples/panel/chat-with-pdf/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,137 @@
...
+annotated-types==0.6.0
+anyio==4.3.0
+asgiref==3.7.2
```
Comment (edublancas) [inline examples/panel/chat-with-pdf/requirements.txt:6]: let's pin versions but only for the explicitly imported packages

**Review by edublancas — CHANGES_REQUESTED** [review]

File: examples/panel/chat-with-pdf/requirements.txt (hunk 2 above)
Comment (LatentDream) [reply examples/panel/chat-with-pdf/requirements.txt]: Fix: [3a3a901](https://github.com/ploomber/doc/pull/284/commits/3a3a9018cab02eeef5c5fe28847481d41929a066)

**Review by edublancas — APPROVED** [review]

**Pull request merged**